main.go

```diff
@@ -167,6 +167,7 @@ func main() {
 	os.Setenv("AUTO_INSTRUMENTATION_JAVA", autoInstrumentationJava)
 	os.Setenv("AUTO_INSTRUMENTATION_PYTHON", autoInstrumentationPython)
 	os.Setenv("AUTO_INSTRUMENTATION_DOTNET", autoInstrumentationDotNet)
+	os.Setenv("AUTO_INSTRUMENTATION_DOTNET", autoInstrumentationNodeJS)
```

Comment:
Could you double check this merge ? 

We need to add the node-js environment variables, this is duplicating the dotnet one
See https://github.com/aws/amazon-cloudwatch-agent-operator/blob/nodejs-instrumentation/main.go#L156-L169 for reference